src/Integration/TeamExplorer/ProjectViewModel.cs

```diff
@@ -18,6 +18,9 @@ internal class ProjectViewModel : ViewModelBase
         private readonly ContextualCommandsCollection commands = new ContextualCommandsCollection();
         private bool isBound;
 
+        // Ordinal comparer should be good enough: http://docs.sonarqube.org/display/SONAR/Project+Administration#ProjectAdministration-AddingaProject
+        public static readonly StringComparer KeyComparer = StringComparer.Ordinal;
```

Comment:
Why is the `KeyComparer` on `ProjectViewModel`? Shouldn't it really be on `ProjectInformation`?